packages/app-store/routing-forms/lib/handleResponse.ts

```diff
@@ -177,15 +177,23 @@ const _handleResponse = async ({
       }
     } else {
       moduleLogger.debug("Dry run mode - Form response not stored and also webhooks and emails not sent");
-      // Create a mock response for dry run
-      dbFormResponse = {
-        id: 0,
-        formId: form.id,
-        response,
-        chosenRouteId,
-        createdAt: new Date(),
-        updatedAt: new Date(),
-      };
+      if (queueFormResponse) {
+        queuedFormResponse = {
```

Comment:
We need to set queuedResponse in case of dry run as well, similar to regular response